exoctk/exoctk_app/app_exoctk.py

```diff
@@ -471,9 +471,32 @@ def contam_visibility():
             except:
                 pass
 
-            # Calculate
-            title = form.targname.data or ', '.join([form.ra.data, form.dec.data])
-            pG, pB, dates, vis_plot, table = vpa.using_gtvt(str(form.ra.data), str(form.dec.data), form.inst.data)
+
+                contamVars['visPA'] = True
+
+                # Make plot
```

Comment:
This block of code should be unindented once so it is inline with the try/except, not under the except loop.